### PR TITLE
refactor: drink

### PR DESCRIPTION
**Pull request by Daanvdplas**
Refactorings based on: https://github.com/r0gue-io/pop-drink/pull/3

### CODE REVIEW

File: runtime/devnet/Cargo.toml [hunk 1]
```diff
@@ -258,8 +258,10 @@ try-runtime = [
...
 # extension.
-metadata-hash = [ "substrate-wasm-builder/metadata-hash" ]
+metadata-hash = ["substrate-wasm-builder/metadata-hash"]
 
 # A convenience feature for enabling things when doing a build
 # for an on-chain release.
-on-chain-release-build = [ "metadata-hash" ]
+on-chain-release-build = ["metadata-hash"]
+
+drink = []
```
Comment (Daanvdplas) [inline runtime/devnet/Cargo.toml:267]: This is causing issues in CI

Comment (Daanvdplas) [reply runtime/devnet/Cargo.toml:267]: taplo and zepter want different formatting here. 

We should see if there is a way to put something in there and bypass the ci checks. 

Already checked whether there were changes that we could make to the taplo.toml file but unsuccesful.

Making changes to either one of those repos sounds a waste of time.

File: pop-api/examples/fungibles/tests.rs [hunk 2]
```diff
@@ -818,13 +843,13 @@ mod utils {
 		PSP22Error::Custom(status_code.to_string())
 	}
 
-	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
-	/// environment and the contract environment.
+	// This is used to resolve type mismatches between the `AccountId` in the quasi testing
```
Comment (chungquantin) [inline pop-api/examples/fungibles/tests.rs:846]:
```suggestion
	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
```
Shouldn't this is `///`?

File: pop-api/examples/fungibles/tests.rs [hunk 3]
```diff
@@ -818,13 +843,13 @@ mod utils {
...
+	// environment and the contract environment.
 	pub(super) fn account_id_from_slice(s: &[u8; 32]) -> AccountId {
 		AccountId::decode(&mut &s[..]).expect("Should be decoded to AccountId")
 	}
 
-	/// Get the last event from pallet contracts.
+	// Get the last event from pallet contracts.
```
Comment (chungquantin) [inline pop-api/examples/fungibles/tests.rs:852]:
```suggestion
	/// Get the last event from pallet contracts.
```
Shouldn't this is `///`?

File: pop-api/examples/fungibles/tests.rs [hunk 4]
```diff
@@ -808,7 +833,7 @@ mod utils {
 		}
 	}
 
-	/// Convert into `PSP22Error::Custom` error type.
+	// Convert into `PSP22Error::Custom` error type.
```
Comment (chungquantin) [inline pop-api/examples/fungibles/tests.rs:836]:
```suggestion
	/// Convert into `PSP22Error::Custom` error type.
```
Shouldn't this is `///`?

File: pop-api/examples/fungibles/tests.rs [hunk 5]
```diff
@@ -818,13 +843,13 @@ mod utils {
 		PSP22Error::Custom(status_code.to_string())
 	}
 
-	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
-	/// environment and the contract environment.
+	// This is used to resolve type mismatches between the `AccountId` in the quasi testing
+	// environment and the contract environment.
```
Comment (chungquantin) [inline pop-api/examples/fungibles/tests.rs:847]:
```suggestion
	/// environment and the contract environment.
```

File: pop-api/examples/fungibles/tests.rs [hunk 6]
```diff
@@ -12,21 +14,44 @@ use pop_api::{
...
+const CHARLIE: AccountId32 = AccountId32::new([3_u8; 32]);
 const AMOUNT: Balance = MIN_BALANCE * 4;
 const MIN_BALANCE: Balance = 10_000;
 const TOKEN: TokenId = 1;
 
 static CONTRACT: LazyLock<ContractBundle> = LazyLock::new(|| BundleProvider::local().unwrap());
 
-/// Deployment and constructor method tests.
+/// Sandbox runtime environment for `devnet` runtime.
+pub struct Sandbox {
```
Comment (chungquantin) [inline pop-api/examples/fungibles/tests.rs:35]: This is clean!

**Review by chungquantin — APPROVED** [review]

File: pop-api/examples/fungibles/Cargo.toml [hunk 7]
```diff
@@ -21,11 +21,7 @@ env_logger = { version = "0.11.3" }
 serde_json = "1.0.114"
 
 # Local
-drink = { path = "../../../../pop-drink/drink" }
-pop-sandbox = { path = "../../../pop-sandbox", default-features = false }
-
-# Substrate
-frame-support = { version = "36.0.0", default-features = false }
+drink = { package = "pop-drink", path = "../../../../pop-drink/crates/pop-drink" }
```
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply pop-api/examples/fungibles/Cargo.toml:24]: Sounds good

File: pop-api/examples/fungibles/tests.rs [hunk 8]
```diff
@@ -12,21 +14,44 @@ use pop_api::{
...
+	ext: TestExternalities,
+}
+
+impl Default for Sandbox {
+	fn default() -> Self {
+		let balances: Vec<(AccountId32, u128)> =
```
Comment (evilrobot-01) [inline pop-api/examples/fungibles/tests.rs:41]: Assume the ext field name cant be changed, so can you add a comment here about initialisation genesis state so its easy for devs to understand what is going on

**Review by evilrobot-01 — COMMENTED** [review]
Just a minor comment, but note that I left a lot on the underlying PR wrt test improvements.

File: runtime/devnet/Cargo.toml (hunk 1 above)
Comment (Daanvdplas) [reply runtime/devnet/Cargo.toml]: Fixed

File: pop-api/examples/fungibles/Cargo.toml (hunk 7 above)
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply pop-api/examples/fungibles/Cargo.toml:24]: Done

**Pull request merged**